README.md

```diff
@@ -1,3 +1,59 @@
 # Bangalore Clojure Group Website
 
-To be updated!
+[Visit the Website](https://bangalore-clj.github.io/)
+
+
+## About
+
+The Bangalore Clojure Group hosts one of the most active (since 2010) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
+
+Our meetups usually have talks by Clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists and functional freaks without any bias.
+
+Please reach out to us if you have any thoughts on what we can do together or how we can help one another.
+
+Ways to reach us:
+- [clojurians slack](https://clojurians.herokuapp.com/): #bangalore-clj channel
+- [Google group](https://groups.google.com/forum/#!forum/bangalore-clj)
+- Email: `bangaloreclojure@gmail.com`
+
+
+## CONTRIBUBTE
```

Comment:
Make it standard case instead of upper case for consistency?

Reply:
Fixed